### PR TITLE
DOC: Improved linux build instructions

### PR DESCRIPTION
**Pull request by lassoan**
A few `cd` commands were missed or duplicated, now commands can be copy-pasted into a terminal one after the other to configure, build, test, and package Slicer.

### CODE REVIEW

File: Docs/developer_guide/build_instructions/linux.md [hunk 1]
```diff
@@ -105,7 +106,13 @@ of build (Debug or Release mode), whether to use system-installed libraries,
 let the build process fetch and compile own libraries, or enable/disable some of
 the software components and functionalities of Slicer.
 
-To obtain a default configuration of the Slicer build project use `cmake`:
+The following folders will be used in the instructions below:
+
+- source folder: ~/Slicer
+- build folder: ~/Slicer-SuperBuild-Debug
+- inner-build folder: ~/Slicer-SuperBuild-Debug/Slicer-build
```
Comment (jcfr) [inline Docs/developer_guide/build_instructions/linux.md:113]:
```suggestion
| Folder          | Path   |
|-----------------|--------|
| **source**      | `~/Slicer` |
| **build**       |  `~/Slicer-SuperBuild-Debug` |
| **inner-build** |  `~/Slicer-SuperBuild-Debug/Slicer-build` |
```

File: Docs/developer_guide/build_instructions/linux.md [hunk 2]
```diff
@@ -136,10 +143,9 @@ cmake -DCMAKE_BUILD_TYPE:STRING=Release ../Slicer
 ## Build Slicer
 
 Once the Slicer build project files have been generated, the Slicer project can
-be build:
+be built by running this command in the build folder:
```
Comment (jcfr) [inline Docs/developer_guide/build_instructions/linux.md:146]:
```suggestion
be built by running this command in the **build** folder
```

File: Docs/developer_guide/build_instructions/linux.md (hunk 1 above)
Comment (jcfr) [reply Docs/developer_guide/build_instructions/linux.md:113]: Rendering:

| Folder          | Path   |
|-----------------|--------|
| **source**      | `~/Slicer` |
| **build**       |  `~/Slicer-SuperBuild-Debug` |
| **inner-build** |  `~/Slicer-SuperBuild-Debug/Slicer-build` |

File: Docs/developer_guide/build_instructions/linux.md [hunk 3]
```diff
@@ -190,23 +196,27 @@ make
 ## Run Slicer
 
 After the building process has successfully completed, the executable file to
-run slicer will be located in `./Slicer-build/Slicer`
+run Slicer will be located in the inner-build folder.
```
Comment (jcfr) [inline Docs/developer_guide/build_instructions/linux.md:199]:
```suggestion
run Slicer will be located in the **inner-build** folder.
```

File: Docs/developer_guide/build_instructions/linux.md [hunk 4]
```diff
@@ -190,23 +196,27 @@ make
...
+The application can be launched by these commands:
+```
+cd Slicer-build
+./Slicer`
+```
 
 ## Test Slicer
 
-After building, run the tests in the  `Slicer-SuperBuild/Slicer-build` directory.
+After building, run the tests in the inner-build folder.
```
Comment (jcfr) [inline Docs/developer_guide/build_instructions/linux.md:209]:
```suggestion
After building, run the tests in the **inner-build** folder.
```

File: Docs/developer_guide/build_instructions/linux.md [hunk 5]
```diff
@@ -190,23 +196,27 @@ make
...
+Type the following (you can replace 4 by the number of processor cores in the computer):
 ```
-cd Slicer-SuperBuild-Debug/Slicer-build
 ctest -j4
 ```
 
 ## Package Slicer
 
-Start a terminal and type the following:
+Start a terminal and type the following in the inner-build folder:
```
Comment (jcfr) [inline Docs/developer_guide/build_instructions/linux.md:218]:
```suggestion
Start a terminal and type the following in the **inner-build** folder:
```

File: Docs/developer_guide/build_instructions/linux.md [hunk 6]
```diff
@@ -105,7 +106,13 @@ of build (Debug or Release mode), whether to use system-installed libraries,
...
 the software components and functionalities of Slicer.
 
-To obtain a default configuration of the Slicer build project use `cmake`:
+The following folders will be used in the instructions below:
+
+- source folder: ~/Slicer
+- build folder: ~/Slicer-SuperBuild-Debug
+- inner-build folder: ~/Slicer-SuperBuild-Debug/Slicer-build
+
+To obtain a default configuration of the Slicer build project, create the build folder and use `cmake`:
```
Comment (jcfr) [inline Docs/developer_guide/build_instructions/linux.md:115]:
```suggestion
To obtain a default configuration of the Slicer build project, create the **build** folder and use `cmake`:
```

**Comment by lassoan** [comment]
Thanks for the review @jcfr. I've accepted all change suggestions. If you approve this version then I'll squash & merge.

**Review by jcfr — APPROVED** [review]

**Pull request merged**